sdk/spring/README.md

```diff
@@ -75,7 +75,6 @@ The following application starters are provided by Spring Cloud Azure under the
 | spring-cloud-azure-starter-active-directory-b2c      | Starter for using Azure Active Directory B2C with Spring Security  |
 | spring-cloud-azure-starter-appconfiguration          | Starter for using Azure App Configuration                          |
 | spring-cloud-azure-starter-cosmos                    | Starter for using Azure Cosmos DB                                  |
-| spring-cloud-azure-starter-data-cosmos               | Starter for using Azure Cosmos DB and Spring Data Cosmos DB        |
```

Comment:
Why delete this?

Reply:
It's not included in the Spring Data support in this release.